packages/shared/src/components/filters/FollowingFilter.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { type ReactElement } from 'react';
+import Link from 'next/link';
+import { useAuthContext } from '../../contexts/AuthContext';
+import UserList from '../profile/UserList';
+import { useFollowingQuery } from '../../hooks/contentPreference/useFollowingQuery';
+import { ContentPreferenceType } from '../../graphql/contentPreference';
+import { FlexCentered } from '../utilities';
+import { checkFetchMore } from '../containers/InfiniteScrolling';
+import { AddUserIcon } from '../icons';
+import { IconSize } from '../Icon';
+import { useLazyModal } from '../../hooks/useLazyModal';
+
+export const FollowingFilter = (): ReactElement => {
+  const { user } = useAuthContext();
+  const { closeModal } = useLazyModal();
+  const queryResult = useFollowingQuery({
+    id: user.id,
+    entity: ContentPreferenceType.User,
+  });
+  const { data, isFetchingNextPage, fetchNextPage } = queryResult;
+  const users = data?.pages.reduce((acc, p) => {
+    p?.edges.forEach(({ node }) => {
+      acc.push(node.referenceUser);
+    });
+
+    return acc;
+  }, []);
+
+  return (
+    <UserList
+      users={users}
+      emptyPlaceholder={
+        <FlexCentered className="flex-col gap-4 px-6 py-10 text-center text-text-tertiary typo-callout">
```

Comment:
Generic: It seems we need to remove some space (see purple parts) that will make it inline with the other tabs:
<img width="812" alt="Screenshot 2024-09-20 at 07 51 31" src="https://github.com/user-attachments/assets/0259cd57-a7d7-4954-9fe5-7108e37969b4">

Reply:
Fixed... had to set a `!p-0` on the modal body element.